static/js/lightbox.js

```diff
@@ -285,6 +274,10 @@ export function initialize() {
     const rendered_lightbox_overlay = render_lightbox_overlay();
     $("body").append(rendered_lightbox_overlay);
 
+    const pan_zoom_control = new PanZoomControl(
```

Comment:
I wasn't super happy with having this object at this top level scope but I couldn't figure out a way to not endlessly recreate the object inside other events, which lead to some pretty bad behavior. If anyone has any thoughts on a better pattern I'd love to hear them.